coucharchive

```diff
@@ -534,6 +597,11 @@ def main():
         '--reuse-db-if-exist', dest='reuse_db_if_exist',
         action='store_true', default=False,
         help='continue restoration even if database exists on target')
+    sub['restore'].add_argument(
+        '--no-queue-control', dest='no_queue_control',
```

Comment:
Sometimes we use "queue", sometimes "run_queue". One day, we could implement our own queue inside coucharchive.

To avoid problems in the future, I propose to use "run_queue" and `--no-run-queue-control` everywhere. What do you think?